### ISSUE TITLE
Grid of posts 3x2

### ISSUE THREAD

**Issue by iamtakashi** [Automattic/blockpatterns#304]
**Category:** Blog, Wireframe

**Tag:** Pattern, Layout

**Screenshot:**
![dotcompatterns wordpress com__p=8435 preview=true query-16-page=2](https://user-images.githubusercontent.com/908665/200635388-4c036e52-2156-44a9-ba26-bedfa82aed9e.png)

**Markup:**
```
<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"7vw","bottom":"7vw"},"margin":{"top":"0px","bottom":"0px"}}},"layout":{"inherit":true,"type":"constrained"}} -->
<div class="wp-block-group alignfull" style="margin-top:0px;margin-bottom:0px;padding-top:7vw;padding-bottom:7vw"><!-- wp:query {"queryId":16,"query":{"perPage":6,"pages":0,"offset":"0","postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"exclude","inherit":false},"displayLayout":{"type":"flex","columns":3},"align":"wide"} -->
<div class="wp-block-query alignwide"><!-- wp:post-template -->
<!-- wp:group {"style":{"spacing":{"blockGap":"1.5rem","padding":{"bottom":"1.5rem"}}}} -->
<div class="wp-block-group" style="padding-bottom:1.5rem"><!-- wp:post-featured-image {"isLink":true} /-->

<!-- wp:post-title {"isLink":true,"style":{"elements":{"link":{"color":{"text":"var:preset|color|foreground"}}},"typography":{"lineHeight":"1.3"}},"fontSize":"medium"} /-->

<!-- wp:post-excerpt {"moreText":"Read more","showMoreOnNewLine":false,"fontSize":"small"} /-->

<!-- wp:post-date {"fontSize":"small"} /--></div>
<!-- /wp:group -->
<!-- /wp:post-template -->

<!-- wp:query-pagination {"paginationArrow":"arrow","layout":{"type":"flex","justifyContent":"space-between"}} -->
<!-- wp:query-pagination-previous /-->

<!-- wp:query-pagination-numbers /-->

<!-- wp:query-pagination-next /-->
<!-- /wp:query-pagination --></div>
<!-- /wp:query --></div>
<!-- /wp:group -->
```

**Preview:** https://dotcompatterns.wordpress.com/?p=8435&preview=true

cc @alaczek @onuro

**Comment by iamtakashi** [comment]
Like #303, Since TT3 's space unit is a lot generous than other themes, I used an absolute value for the block spacing. Does it look better in TT3 now?

**Comment by iamtakashi** [comment]
Published as it's very similar to #303 that has ben approved. If you see any issues, please let me know.
